Documentation/install/upgrade.rst

```diff
@@ -243,6 +243,18 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 
 * Cilium has bumped the minimal kubernetes version supported to v1.11.0.
 
+* The ``kubernetes.io/cluster-service`` label has been removed from the Cilium
+  `DaemonSet`. Existing users must remove this label from their `DaemonSet`
+  specification to safely upgrade.
+
+  *Required action:*
+
+  .. code:: bash
+
+     $ kubectl get ds -n kube-system cilium -o yaml > cilium-ds.yaml
+     $ sed -i '/kubernetes.io\/cluster-service: "true"/d' cilium-ds.yaml
+     $ kubectl apply -f cilium-ds.yaml --force
+
 * If ``kvstore`` is setup with ``etcd`` **and** TLS is enabled, the field name
```

Comment:
This section needs to be backported to 1.6 branch

Reply:
I'll set a reminder to prepare that backport. The corresponding PR wasn't backported yet AFAIK (https://github.com/cilium/cilium/pull/9519)